examples/vite-svelte/README.md

```diff
@@ -0,0 +1,48 @@
+# Svelte + TS + Vite
```

Comment:
We should not include README and assets (favicons, etc.) in the examples.

Reply:
Yeah, I need to change some internals. 

Maybe we need also exclude some entries used by svelte on 'directives' like transition and animate since vscode and intellij dont work.